desktop_version/src/Script.cpp

```diff
@@ -2114,25 +1798,10 @@ void scriptclass::run(void)
 			}
 			else if (words[0] == "createlastrescued")
 			{
-				if (game.lastsaved==2)
-				{
-					r=14;
-				}
-				else if (game.lastsaved==3)
-				{
-					r=15;
-				}
-				else if (game.lastsaved==4)
-				{
-					r=13;
-				}
-				else if (game.lastsaved==5)
-				{
-					r=16;
-				}
-				else
+				r = graphics.crewcolour(game.lastsaved);
+				if (r == 1 || r == PURPLE)
```

Comment:
Not sure what this `r == 1` here. `graphics.crewcolour()` will return either 0, 20, 14, 15, 13, or 16. So this doesn't blacklist Viridian's color.

Reply:
oh my god i meant to write 0